tests/src/ViewModel/ButtonCollectionTest.php

```diff
@@ -20,13 +20,15 @@ public function it_has_data()
                     'text' => 'text',
                 ],
             ],
+            'inline' => true,
             'centered' => true,
             'compact' => true,
         ];
 
-        $blockLink = new ButtonCollection([Button::link($data['buttons'][0]['text'], $data['buttons'][0]['path'])], $data['centered'], $data['compact']);
+        $blockLink = new ButtonCollection([Button::link($data['buttons'][0]['text'], $data['buttons'][0]['path'])], true, $data['centered'], $data['compact']);
```

Comment:
Change to: 

```
$blockLink = new ButtonCollection([Button::link($data['buttons'][0]['text'], $data['buttons'][0]['path'])], $data['inline'], $data['centered'], $data['compact']);
```

Reply:
The parameter is now refactored to $data['inline'].